### DISCUSSION TITLE
Data backup

### DISCUSSION THREAD

**Discussion by shinebrillant** [enrique-lozano/Monekin#175] [Ideas / New features]
Current Behavior
When backing up, select .db and .csv to export respectively, and then move .db and .csv to other folders in android/download for backup.

Expected Behavior
Customize .db and .csv file names, customize export paths, customize export types, and automatically export.

**Comment by enrique-lozano** [comment]
Hmmm, I like the idea of an automatic export but I think this could potentially filling up space on your device without realizing it, especially for more basic users. Also, for some reason, no competing app does this locally, so I think there must be more problems than it seems. Some do make an automatic copy to Google Drive (many of them under premium subscription, I don't know if the API will have any cost).

What I have considered and would like to do is put a button inside the snackbar that opens saying that the file has been downloaded correctly. This new button could take you to the directory of that file.

**Comment by shinebrillant** [comment]
`Hmmm, I like the idea of an automatic export but I think this could potentially filling up space on your device without realizing it, especially for more basic users.`

Users can choose to turn on or off.

`Also, for some reason, no competing app does this locally, so I think there must be more problems than it seems.`

I don't understand what it means.

`Some do make an automatic copy to Google Drive (many of them under premium subscription, I don't know if the API will have any cost).`

I use Nextcloud, and I can upload the specified folder. As long as Monkin has the function of exporting the selection folder and automatic export, it can be easily managed.

`What I have considered and would like to do is put a button inside the snackbar that opens saying that the file has been downloaded correctly. This new button could take you to the directory of that file.`

This is great. :tada:

**Comment by enrique-lozano** [comment]
Closing this as we are tracking the auto-local-backups feature in #268 and in #257
